src/PresentationalComponents/Inventory/Inventory.js

```diff
@@ -68,11 +68,35 @@ const Inventory = ({
           {}
         )
       )?.data.data;
-      const issues = pathways.map((rec) => ({
-        id: `advisor:${rec.rule_id}`,
-        description: rec.description,
-      }));
-      return { issues, systems: selected };
+
+      const systems = (
+        await Get(
+          `${BASE_URL}/pathway/${encodeURI(pathway.slug)}/reports/`,
+          {},
+          {}
+        )
+      )?.data.rules;
+
+      let issues = [];
```

Comment:
This could be constructed using `.find`, or?

Reply:
Not sure how, can you make code proposal?

Reply:
@MichalSajdik I may have meant `.filter` or a `.map`, to avoid the `.forEach`s. Though, I'm not sure if it would make this better in any aspect.

Reply:
Yea, I would be able to somehow rewrite this using these function, but I doubt that it would look better